charts/datadog/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Datadog changelog
 
+## 2.37.2
+
+* Set traced_cgroups_count default value to 0.
```

Comment:
Could you mentioned that this is a CWS config in system probe? thanks!

Reply:
sure, I updated the message